File: Programming Fundamentals/Labs And Homeworks/ArraysLab/EvenAndOddSubtraction.java
package arraysLab;

import java.util.Arrays;
import java.util.Scanner;

public class EvenAndOddSubtraction {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] numbersArr = Arrays
                .stream(scanner.nextLine().split(" "))
                .mapToInt(Integer::parseInt)
                .toArray();
        int evenSum = 0;
        int oddSum = 0;
        for (int item : numbersArr) {
            if (item % 2 == 0) {
                evenSum += item;
            } else {
                oddSum += item;
            }

        }
        System.out.println(evenSum - oddSum);

    }

}

File: JS Front-End/JS Syntax Fundamentals - Lab/evenAndOddSubtraction.js
function evenAndOddSubtraction(array) {
    let oddSum = 0;
    let evenSum = 0;
    array.filter(e => e % 2 === 0).forEach(e => evenSum += e);
    array.filter(e => e % 2 != 0).forEach(e => oddSum += e);
    console.log(evenSum - oddSum);
}

evenAndOddSubtraction([3, 5, 7, 9]);